js/base/main.js

```diff
@@ -296,12 +303,16 @@ export class RipeCommonsMainPlugin extends RipeCommonsPlugin {
             // updates the store with both the groups and the supported
             // characters of the current configuration context
             this.store.commit("initialsGroups", groups);
+            this.store.commit("initialsMinCharacters", minimumCharacters);

```

Comment:
Same concept = same name.

```suggestion
            this.store.commit("minimumCharacters", minimumCharacters);
```